joomla/joomla-cms
# [4.0] [plg_system_stats] Move fields to subdirectory

### Summary of Changes

Moves fields of `plg_system_stats` to `src` subdirectory to avoid case sensitivity issues when upgrading from J3.

### Testing Instructions

Apply patch. Delete `libraries/autoload_psr4.php` file.
Edit System - Joomla! Statistics plugin. Check that `Select here to see the information that will be sent.` text and `Reset Unique ID` button still appear.

### Documentation Changes Required

No.
